aptos-move/aptos-vm/src/move_vm_ext/respawned_session.rs

```diff
@@ -289,7 +298,7 @@ impl<'r> TResourceGroupView for ExecutorViewWithChangeSet<'r> {
         resource_tag: &Self::ResourceTag,
         maybe_layout: Option<&Self::Layout>,
     ) -> anyhow::Result<Option<Bytes>> {
-        // TODO: resource_group_write_set also contains a layout. What to do with it?
+        // TODO[agg_v2](test): resource_group_write_set also contains a layout. What to do with it?
```

Comment:
I don't think we need to do something special?

Reply:
I added function to check the layouts match. But not sure how to do this check sporadically.